modules/userId/index.js

```diff
@@ -570,8 +570,8 @@ export function init(config) {
   }
   // listen for config userSyncs to be set
   config.getConfig(conf => {
-    // Note: support for both 'userSync' and 'usersync' will be deprecated with Prebid.js 3.0
-    const userSync = conf.userSync || conf.usersync;
+    // Note: support for 'usersync' was dropped as part of Prebid.js 4.0
+    const userSync = conf.userSync;
```

Comment:
LGTM, I confirmed no doc changes needed to http://prebid.org/dev-docs/publisher-api-reference.html#setConfig-Configure-User-Syncing

Reply:
actually can you change the tests at test/spec/modules/userId_spec.js that have config usersync ?

Reply:
also the examples at https://github.com/prebid/Prebid.js/blob/master/modules/userId/userId.md

Reply:
also here https://github.com/prebid/Prebid.js/blob/master/modules/userIdTargeting.md